templates/kafka/0/docker-compose.yml

```diff
@@ -0,0 +1,16 @@
+zk:
```

Comment:
@vincent99. I'm completely agree with you. 

As i said before, i understand very well your point. I would shared it completely, if you already have a nested catalog that it could deploy an already defined service. When i started to think about how to build the package, i thought to do it exactly as you said, but i decided to change my mind, although i know it is a worse design , due to the missing feature that you say of the nested catalog. 

Are you working in that way?? Is in you backlog this feature?? You make so happy if you say that yes. :)

If this feature is coming on, i have no problem to do that you suggest. I do a new PR to your rancher-catalog zookeeper and i do an easy modification in confd templates for broker and push it again. 

Right now, zk cluster and broker clusters, can run in a dynamic or in a static mode. By default, they run in dynamic mode (you could scale up and down the cluster dinamically and separately), checking every $CONFD_INTERVAL seconds (60 by default) for new config, but you if you set $CONFD_INTERVAL to 0 they run in static mode (you have to reboot the clusters to scale). Do you think that the possibility to choose between the 2 run modes, is a valuable feature??